src/pages/Overview/NoBridgeFeesBanner.tsx

```diff
@@ -14,16 +14,16 @@ export const NoBridgeFeesBanner: FC<AnnouncementBannerContainerProps> = ({
   return (
     <AnnouncementBannerContainer
```

Comment:
As suggested in <ID HERE>:

```suggestion
    <Card
        borderWidth={0}
```

Remember to update props typings - you can use `BoxProps` since the `Card` components is a wrapper for the `Box` component and doesn't export the `CardProps` type itself.

As a side note - perhaps the box shadow should be disabled as well. There is an inconsistency in this area between the implementation and Figma designs.

Reply:
Thanks. Implemented a solution in bc56dbc1035d2e36466ed003b51bee3d6bfdebb4. I've also removed the box shadow.

Ref: https://github.com/threshold-network/token-dashboard/pull/711#discussion_r1437094324